services/static-webserver/client/source/class/osparc/NewRelease.js

```diff
@@ -36,7 +36,11 @@ qx.Class.define("osparc.NewRelease", {
       this._add(introLabel);
 
       const detailsText = this.tr("What's new");
-      const link = osparc.utils.LibVersions.getVcsRefUrl();
+      let link = osparc.utils.LibVersions.getVcsReleaseUrl();
```

Comment:
@odeimaiz this cannot be obtained from the osparc.envs since those are set while building the image and at that time there is no information about the release this image might be.

This information can be retrieved instead either from the ``/config`` or ``/static-frontend-data.json``